webapp/src/test/java/com/box/l10n/mojito/smartling/SmartlingClientTest.java

```diff
@@ -10,10 +10,7 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.io.ByteStreams;
-import org.junit.Assert;
```

Comment:
let's avoid wildcard import. To configure Intellij: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports